packages/neuron-wallet/src/controllers/sudt.ts

```diff
@@ -32,5 +33,18 @@ export default class SUDTController {
       result: {tokenID: params.tokenID, symbol: symbol, tokenName: name, decimal: decimal},
     }
   }
+
+  public getSUDTTypeScriptHash(params: { tokenID: string }): Controller.Response<string> {
```

Comment:
This calculation could be done in the renderer process so that the main process can handle other things at the same time.

Reply:
so i should add a const `sudt.codeHash` in the renderer process?

Reply:
> so i should add a const `sudt.codeHash` in the renderer process?

I see, it's ok to fetch the result if `code hash` and `hash type` are missing in the renderer process.